megamek/src/megamek/common/net/DataStreamConnection.java

```diff
@@ -27,7 +27,7 @@
  * <code>DataInputStream</code> and <code>DataOutputStream</code> to
  * send/receive data.
  */
-class DataStreamConnection extends AbstractConnection {
+class DataStreamConnection extends AbstractSocketConnection {
```

Comment:
I think this may have been a holdover from when I was considering inserting an abstraction at this level  -- can be removed if it's an issue